src/field.rs

```diff
@@ -177,12 +170,9 @@ impl FieldElement {
     ///
     /// The inverse is computed as self^(p-2), since
     /// x^(p-2)x = x^(p-1) = 1 (mod p).
-    //
-    // XXX do we want the debug assertion to check for zero? it breaks behaviour
-    // such as that such as in curve25519_dalek::montgomery::test::identity_to_monty.
+    ///
+    /// This function returns zero on input zero.
```

Comment:
Not sure if this should be in a `# Warning` section, or at least a different section? Up to you.

Reply:
It's only in internal docs, so I don't think it really matters.